libkbfs/md_journal.go

```diff
@@ -1178,7 +1226,11 @@ func (j *mdJournal) clear(
 	// Garbage-collect the old branch entries.  TODO: we'll eventually
 	// need a sweeper to clean up entries left behind if we crash
 	// here.
+	isLocalSquash := bid == LocalSquashBranchID
```

Comment:
It seems like you only want to clear out non-squash entries if bid = LocalSquashBranchID, right? If so, I think you missed the call to `j.j.clear()` above -- it should also take a bid, right?